app/scripts/lib/auth-errors.js

```diff
@@ -17,6 +17,7 @@ define(function (require, exports, module) {
   };
 
   var UNEXPECTED_ERROR_MESSAGE = t('Unexpected error');
+  const EMAIL_BOUNCED_MESSAGE = t('Your verification email was just returned');
```

Comment:
The bounced email could come from other templates besides verification email. Should we exclude `verification` messaging and be a little more vague?